pkgs/applications/networking/sync/rclone/default.nix

```diff
@@ -1,6 +1,6 @@
 { lib, stdenv, buildGoModule, fetchFromGitHub, buildPackages, installShellFiles
 , makeWrapper
-, enableCmount ? true, fuse, macfuse-stubs
+, enableCmount ? true, fuse, fuse3, macfuse-stubs
```

Comment:
Shall we also change it here?
```suggestion
, enableCmount ? true, fuse3, macfuse-stubs
```

Reply:
I tried that but then the cgo fuse part is no longer compiling. See the link to the workflow, too.